### PR TITLE
Replace pato:towards and pato:correlates_with object properties with RO ones

### PR DESCRIPTION
**Pull request by shawntanzk**
related to #454

### CODE REVIEW

File: src/ontology/pato-edit.obo [hunk 1]
```diff
@@ -22478,8 +22484,3 @@ name: singly_occurring_form_of
 comment: PATO divides qualities between normal (monadic, singly-occurring) qualities and relational qualities. Relational qualities stand in the 'towards' relation with respect to some additional entity. For example, The sensitivity of an eye towards red light. In some cases we want to represent a quality such as 'protruding' in both monadic and relational branches. We use this relation to link them.
 synonym: "monadic_form_of" EXACT []
 
-[Typedef]
-id: towards
```
Comment (matentzn) [inline src/ontology/pato-edit.obo:22482]: Shouldnt this be obsoleted properly?

**Review by matentzn — CHANGES_REQUESTED** [review]
you dont need the module in this PR anymore - the base migration should take care of that. Just include the changes to pato-edit.obo.

Get @dosumis to take a look, but should be fine.

File: src/ontology/pato-edit.obo (hunk 1 above)
Comment (shawntanzk) [reply src/ontology/pato-edit.obo:22482]: Did not obsolete, but instead replaced it by adding a RO ID on it - happy to properly obsolete this too if we should do it that way

**Comment by shawntanzk** [comment]
> you dont need the module in this PR anymore - the base migration should take care of that. Just include the changes to pato-edit.obo.

Shouldn't we still import terms properly even with the base file stuff? Or do we just let the release handle that part?

**Review by nicolevasilevsky — APPROVED** [review]

**Comment by shawntanzk** [comment]
> 2\. With the new Base process, it will now take much longer to generate a module - but maybe you should try:

```
shawntan@C02DW26DML87 ontology % sh run.sh make refresh-merged
pato.Makefile:11: warning: overriding recipe for target 'pato.obo'
Makefile:429: warning: ignoring old recipe for target 'pato.obo'
make IMP=true IMP_LARGE=true MIR=true PAT=false imports/merged_import.owl -B
make[1]: Entering directory '/work/src/ontology'
pato.Makefile:11: warning: overriding recipe for target 'pato.obo'
Makefile:429: warning: ignoring old recipe for target 'pato.obo'
make[1]: *** No rule to make target 'imports/merged_import.owl'.  Stop.
make[1]: Leaving directory '/work/src/ontology'
make: *** [Makefile:254: refresh-merged] Error 2
```

Should I instead refresh imports and make a release? @matentzn

**Comment by matentzn** [comment]
You have to merge master in your branch first I would assume.

**Comment by shawntanzk** [comment]
Currently error due to decprecated towards - think its a mistake when merging - will fix this after computer runs refresh-merge (trying to run it now as the "airplane" noise is bad for my sleep if i run it overnight >.<)

**Comment by shawntanzk** [comment]
ok whole mess, can't deal, going to close this and redo this whole thing >.<

**Pull request closed without merging**